### DISCUSSION TITLE
How to import existing file and folder structure?

### DISCUSSION THREAD

**Discussion by DominikBritz** [redimp/otterwiki#132] [Q&A] [answered]
I want to import my existing markdown files, which are structured in folders. 

How can I import them to the repository?

**Comment by redimp** [comment, marked as answer]
Hey @DominikBritz,

Yes! the most convenient way to do this is via git:

1. Enable the Git Webserver via `Settings > Application Preferences > Content end Editing Preferences > [x] Enable Git Webserver`
2. Depending on the structure you have, yu might want to set `[x] Retain page name case`, too.
3. Clone the (empty) Otterwiki locally, `git clone https://otterwiki.example.com/.git` -- You find the url on your Settings page.
4. Copy your existing files and directories into the local repo, run `git add`, `git commit` and finally `git push`.

Another way would be to just copy the files into the directroy volume (when running in docker) and running `git add` and `git commit` as user www-data/uid 33.
